src/components/Search.js

```diff
@@ -331,6 +332,11 @@ class Search extends Component {
         try {
             const inputRef = get(searchRef, 'current._inputRef', null);
 
+            if(this.userId) {
```

Comment:
@lohi212 We don't want to set `userId` if it's not defined. Since the above fallback sets it to `''`, this condition will set the empty string as userId.

Instead, explicitly check that it's both not an empty string and not null.